customer_outstanding_statement/views/statement.xml

```diff
@@ -0,0 +1,206 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2018 Eficent Business and IT Consulting Services S.L.
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+<odoo>
+    <template id="customer_outstanding_statement.statement_document">
+        <t t-call="web.external_layout">
+            <div class="page">
+                <div class="row">
+                    <div class="col-xs-5 col-xs-offset-7">
+                        <span t-field="o.name"/><br/>
+                        <span t-raw="o.contact_address.replace('\n\n', '\n').replace('\n', '&lt;br&gt;')"/>
+                        <span t-field="o.vat"/>
+                    </div>
+                    <h4 style="padding-left:20em">
+                        Outstanding Statement
+                    </h4>
+                    <p>
+                        Date: <span t-esc="Date[o.id]" /><br/>
+                        <t t-if="o.ref">Partner ref: <span t-field="o.ref"/></t>
+                    </p>
+
+                    <t t-if="Lines[o.id]">
+                        <br/>
+                        <t t-foreach="Lines[o.id]" t-as="currency">
+                            <br t-if="not currency_first" />
+                            <p>
+                                Outstanding Statement in <span t-esc="Currencies[o.id][currency].name"/>:
```

Comment:
@hveficient can you put gere the end date, then to have "Outstanding statement at" date_end "in currency".

Reply:
It's repeating because you can see the date_end at the last row, but fine by me.